### PR TITLE
Changed parameterization of FMI.ExportContainers examples

### LINKED ISSUE #1540: Correct parameterization in FMI.ExportContainers.Examples.FMUs.HVACZone
This issue is to correct the parameterization in 
`Fluid/FMI/ExportContainers/Examples/FMUs/HVACZone.mo`
and
`Fluid/FMI/ExportContainers/Examples/FMUs/HVACZones.mo`

This integrates the changes of https://github.com/lbl-srg/modelica-buildings/issues/2624 in the IBPSA library.

### PR DESCRIPTION
**Pull request by mwetter**
This closes #1540

### CODE REVIEW

File: IBPSA/Fluid/FMI/ExportContainers/Examples/FMUs/HVACZones.mo [hunk 1]
```diff
@@ -43,18 +47,18 @@ block HVACZones
...
   // Water temperatures and mass flow rates
-  parameter Modelica.SIunits.Temperature TWSup_nominal = 273.15+16
+  parameter Modelica.SIunits.Temperature TWSup_nominal = 285.15
     "Water supply temperature";
-  parameter Modelica.SIunits.Temperature TWRet_nominal = 273.15+12
+  parameter Modelica.SIunits.Temperature TWRet_nominal = 289.15
     "Water return temperature";
   parameter Modelica.SIunits.MassFlowRate mW_flow_nominal=
-    QCoiC_flow_nominal/(TWRet_nominal-TWSup_nominal)/4200
+    -QCoiC_flow_nominal/(TWRet_nominal-TWSup_nominal)/4200
```
Comment (cvering) [inline IBPSA/Fluid/FMI/ExportContainers/Examples/FMUs/HVACZones.mo:61]: Hey Michael, thanks for making the pull request. I checked out the code changes in the review and it looks alright. Can you please check if the sign change is exactly as intended here? If you confirm this change, we can do the merge into the master.

**Review by cvering — CHANGES_REQUESTED** [review]
Hey @mwetter , thanks for merging the new code into the master. Would you please doublecheck my comment below? All the best

File: IBPSA/Fluid/FMI/ExportContainers/Examples/FMUs/HVACZones.mo (hunk 1 above)
Comment (mwetter) [reply IBPSA/Fluid/FMI/ExportContainers/Examples/FMUs/HVACZones.mo:61]: This sign is correct and leads to `mW_flow_nominal` to be positive.

**Review by cvering — APPROVED** [review]
@mwetter Thanks for the update. The pull request is approved, now.

**Pull request merged**